config/aws.spc

```diff
@@ -27,6 +27,14 @@ connection "aws" {
   # from an AWS credential file with the `profile` argument:
   #profile = "myprofile"
 
+  # If no credentials are specified, the plugin will use the AWS credentials
```

Comment:
Is this example still correct, or does it follow the nested structure now?

Reply:
@cbruno10 Thank you, I had forgot to update the example after changing to a map for configuration. The example has now been updated.